### DISCUSSION TITLE
[ERR_STREAM_DESTROYED]: Cannot call write after a stream was destroyed

### DISCUSSION THREAD

**Discussion by luocong2016** [coder/code-server#4550] [Q&A]
#### ENV
```
linux
docker
code-server
```

#### BUILD
```
mkdir /home/coder

sudo chmod 777 /home/coder

docker run -it -p 8088:8080 --name code-server -e PASSWORD=1234 -v "/home/coder:/home/coder" -d codercom/code-server:latest
```

#### ERR
```
[ERR_STREAM_DESTROYED]: Cannot call write after a stream was destroyed
```

**Comment by jsjoeio** [comment]
Is code-server missing permissions or something? This is what [S/O says](https://stackoverflow.com/a/56322721/3015595): 
> grant wrx access to log files, like chmod 777.

Which I guess you're doing here: 
> sudo chmod 777 /home/coder

Hmm 🤔 Is there a stack trace attached to that ERR? I wonder if that would give more information
